about.html

```diff
@@ -0,0 +1,125 @@
+      professionalism, and care. Our values of fairness, progress, focus on people, results, care and responsibility, and effective 
+      service delivery are reflected in everything we do, from our innovative curriculum to our exceptional teaching staff. 
+      We are proud to offer a supportive and inclusive community that values diversity, creativity, and environmental responsibility. 
+      We invite you to explore our website and learn more about how Royal Primary Secondary School can help your child reach their full potential.</p>
+      </div>
+      <div class="dotted-line">
+      <img src="./images/Rectangle-55.svg" alt=""> <img src="./images/Rectangle-55.svg" alt=""> <img src="./images/Rectangle-55.svg" alt="">
+      </div>
+    <div class="key-details"><p></p></div>
+  </div>
+  <div class="small-note">Term 2 Dates: May 12th - July 24th 2023. Mon - Fri, 08:00-15:30</div>
+  <div class="about-royal-logo">
+    <div class="royal-logo-title"><p>Royal School Logo</p></div>
+    <div class="title-line"></div>
+
+    <div class="royal-logo-desc"><p>Royal Primary & Secondary School's logo features gold and purple. Gold represents excellence, achievement, 
+      and prestige<span class="desk-only">, reflecting our commitment to upholding the highest standards of 
+      academic and personal excellence</span>. Purple represents wisdom, creativity, and imagination <span class="desk-only">, reflecting our commitment to 
+      fostering a supportive and innovative learning environment that encourages students to explore new ideas and develop 
+      their own unique talents. Together, these colors embody the spirit of Royal School and our dedication to helping our 
+      students succeed and thrive.</span></p>
+      </div>
+    <div><img class="logo-img" src="./images/School-logo.jpeg" alt="Royal school logo">
+    </div>
+  </div>
+
+  <div class="past-term-highlights">
+    <div class="past-term-title"><h3>View Past Term Highlights</h3>
+    </div>
+    <div class="title-line"></div>
+    <div class="term-highlights-desc"><p>Take a look at a few highlights over our past years</p>
+    </div>
+    <div class="past-img">
+
+      <div class="highlight high1">
+        <h2>2022</h2>
+        <p>First Grade 12 Graduation ceremony</p>
+      </div>
+      <div class="highlight high2">
+        <h2>2021</h2>
+        <p>Commissioning of New Science Classroom</p>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+<section id="about-partners">
+  <div class="school-values-title partners-title"><h4>Partners</h4></div>
+  <div class="title-line"></div>
+
+    <div class="partners-div">
+    <img class="partners" src="./images/mozilla.png" alt="">
+    <img class="partners" src="./images/google.png" alt="">
+    <img class="partners" src="./images/naver.png" alt="">
+    <img class="partners kakao" id="" src="./images/kakao.png" alt="">
+    <img class="partners airbnb" id="" src="./images/airbnb.png" alt="">
```

Comment:
- [ ] For accessibility purposes, it is required to include `alt` attributes for almost all the images used in your project. The `alt` attribute provides alternative text for the image, which is important for users who are visually impaired and use screen readers to navigate the web. Without an `alt` attribute, these users will not be able to understand the content of the image. Therefore, it is important to provide descriptive and accurate `alt` text for all images used in your project to ensure accessibility for all users. Kindly check the images inside your index.html and about.html page :+1: :100:.